lib/game_manager.rb

```diff
@@ -102,6 +102,37 @@ def favorite_opponent(id)
    @numbs = teams.min_by { |v| freq[v] }
  end
 
+ def rival(id)
```

Comment:
This is definitely gonna need some refactoring. I'll probably end up putting this into a module